src/Ev.ServiceBus.Abstractions/Listeners/BaseArgs.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Azure.ServiceBus;
+
+namespace Ev.ServiceBus.Abstractions
+{
+    public class BaseArgs

```

Comment:
```ExecutionBaseArgs``` maybe?

BaseArgs is too generic in my opinion

Reply:
Done